saleor/webhook/response_schemas/taxes.py

```diff
@@ -0,0 +1,36 @@
+from decimal import Decimal
+from typing import Annotated
+
+from pydantic import BaseModel, Field, ValidationInfo, field_validator
+
+from ...core.prices import MAXIMUM_PRICE
+
+
+class LineCalculateTaxesSchema(BaseModel):
+    tax_rate: Annotated[Decimal, Field(ge=0, le=100)]
+    total_gross_amount: Annotated[Decimal, Field(ge=0, le=MAXIMUM_PRICE)]
+    total_net_amount: Annotated[Decimal, Field(ge=0, le=MAXIMUM_PRICE)]
+
+
+class CalculateTaxesSchema(BaseModel):
+    shipping_tax_rate: Annotated[Decimal, Field(ge=0, le=100)]
+    shipping_price_gross_amount: Annotated[Decimal, Field(ge=0, le=MAXIMUM_PRICE)]
+    shipping_price_net_amount: Annotated[Decimal, Field(ge=0, le=MAXIMUM_PRICE)]
+    lines: list[LineCalculateTaxesSchema] = []
+
+    @field_validator("lines")
+    @classmethod
+    def validate_lines_length(
```

Comment:
It is only suggestion, but as for `ExcludedShippingMethodSchema` we have `clean_id` for field_validator, then maybe worth to keep the same naming here?  - like `clena_lines` or `clean_lines_length`, or `ExcludedShippingMethodSchema.validate_id`